parser.y

```diff
@@ -444,17 +444,17 @@ rule
            builder.add_rhs(token)
            result = builder
          }
-     | rhs symbol parameterizing_suffix TAG?
+     | rhs symbol parameterizing_suffix named_ref_opt TAG?
```

Comment:
Could you add test cases for this patten?

Reply:
Thank you! Add test case: https://github.com/ruby/lrama/pull/414/commits/effe94adab3babaa1646d94ae4726cd5246a86fd